flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/DisplayUnit.js

```diff
@@ -5,6 +5,7 @@ class DisplayUnit {
         this.isPowered = isPoweredFn;
         this.potentiometerId = potentiometerId;
 
+        this.systemsInit = Boolean(SimVar.GetSimVarValue('L:A32NX_COLD_AND_DARK_SPAWN', 'Bool') && SimVar.GetSimVarValue('L:A32NX_FMGC_FLIGHT_PHASE', 'Enum') === (FmgcFlightPhases.PREFLIGHT));
```

Comment:
**issue**: `systemsInit` not descriptive.
**issue**: Lots of additional code for what can be a change to the expression setting `this.offDurationInMilliseconds`.

```suggestion
        const initialisingColdAndDark = SimVar.GetSimVarValue('L:A32NX_COLD_AND_DARK_SPAWN', 'Bool') && SimVar.GetSimVarValue('L:A32NX_FMGC_FLIGHT_PHASE', 'Enum') === FmgcFlightPhases.PREFLIGHT;
        this.offDurationInMilliseconds = initialisingColdAndDark ? DisplayUnitSelfTest.RequiredAfterBeingOffForMilliseconds : 0;
```

Reply:
First issue resolved
Second issue, the code is not superflous, ``this.offDurationInMilliseconds += deltaTime;`` must be escaped